adminsortable/admin.py

```diff
@@ -297,8 +297,11 @@ def do_sorting_view(self, request, model_type_id=None):
 
                 for index in indexes:
                     obj = objects_dict.get(index)
-                    setattr(obj, order_field_name, start_index)
-                    obj.save()
+                    # perform the update iff the order field has changed

```

Comment:
I think in this case, using "if" is clearer as to the intention. I think the code looks fine - wish I'd thought of that :) Nice work.